src/drive/mobile/modules/upload/index.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react'
+import localforage from 'localforage'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+import PropTypes from 'prop-types'
+
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import { Query, cancelable, withClient } from 'cozy-client'
+
+import { translate } from 'cozy-ui/transpiled/react'
+import { Modal } from 'cozy-ui/transpiled/react'
+
+import { uploadFilesFromNative } from 'drive/web/modules/upload'
+
+import { ROOT_DIR_ID, TRASH_DIR_ID } from 'drive/constants/config'
+import Header from 'drive/web/modules/move/Header'
+import Explorer from 'drive/web/modules/move/Explorer'
+import FileList from 'drive/web/modules/move/FileList'
+import Loader from 'drive/web/modules/move/Loader'
+import LoadMore from 'drive/web/modules/move/LoadMore'
+import Footer from 'drive/web/modules/move/Footer'
+import Topbar from 'drive/web/modules/move/Topbar'
+
+export const generateForQueue = files => {
+  const filesForQueue = []
+  files.map(file => {
+    filesForQueue.push({ file: file, isDirectory: false })
+  })
+  return filesForQueue
+}
+
+export class DumbUpload extends Component {
+  state = {
+    items: null,
+    folderId: ROOT_DIR_ID
+  }
+  async componentDidMount() {
+    const items = await localforage.getItem('importedFiles')
+    this.setState({ items })
+  }
+
+  async uploadAndState() {
```

Comment:
I don't understand the `AndState` in the name, what does it mean?

Reply:
Me neither. At first I had to setter a state maybe...